### DISCUSSION TITLE
How to cancel the style of ListboxItem component when it is focused?

### DISCUSSION THREAD

**Discussion by iGmainC** [heroui-inc/heroui#3391] [Help] [answered]
How to cancel the style of ListboxItem component when it is focused?

![image](https://github.com/nextui-org/nextui/assets/48195631/40e882a7-3099-4e37-957a-4a703b3fe183)

![image](https://github.com/nextui-org/nextui/assets/48195631/6171a20b-f36f-4abd-b114-e9db8784e874)

![image](https://github.com/nextui-org/nextui/assets/48195631/f4f70159-5dbb-4750-ab15-488a63657df7)

**Comment by wingkwong** [comment, marked as answer]
you can pass your custom classes to `classNames.base`. e.g.

```tsx
classNames={{
    base: "data-[selectable=true]:focus:bg-primary data-[selectable=true]:focus:text-primary-foreground"
}}
```

**Reply by iGmainC** [reply]
thank you！
